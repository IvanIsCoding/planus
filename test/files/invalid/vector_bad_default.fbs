table Foo {
  x: [bool] = [true];
}