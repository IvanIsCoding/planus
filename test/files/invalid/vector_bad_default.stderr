[0m[1m[38;5;9merror[0m[1m: Unsupported default value[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/vector_bad_default.fbs:2:15
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: [bool] = [0m[31m[true][0m;
  [0m[34mâ”‚[0m               [0m[31m^^^^^^[0m [0m[31mVectors only support default values of null or the empty list[0m

Error: Bailing because of errors
